---
title: Dr. Jan Taubenheim
subtitle: Computational biologist working in host microbe interactions with a specific interest in life-history decisions and phenotypic plasticity.
output: 
    prettydoc::html_pretty: 
        theme: cayman
        highlight: github
        use_bookdown: TRUE
        toc: TRUE
#bibliography: ["/home/jan/Work/Administration/Homepage/Personal/Porthmeus.github.io/MyPublications.bib"]
bibliography: ["../dat/MyPublications.bib"]
nocite: '@*'
csl: "../dat/citationStyle.csl"
link-citations: true
---


```{r knitrSetup, include=FALSE, echo = FALSE}
knitr::opts_chunk$set(warning=FALSE, message=FALSE, fig.width = 10, fig.height=5)
#knitr::read_chunk("path/to/chunks")
```


# About me

:::::: {.cols data-latex=""}

::: {.col data-latex="{0.55\textwidth}"}
```{r portrait,echo = FALSE,out.height = "175px",out.width = "137px", out.extra='style = "float: left;margin-right: 10px;margin-top: 40px;"'}
knitr::include_graphics("../img/IMG_3192_pp.jpg")
```

:::

::: {.col data-latex="{0.05\textwidth}"}
\ 
<!-- an empty Div (with a white space), serving as
a column separator -->
:::

::: {.col data-latex="{0.4\textwidth}"}

I am postdoc in the Kaleta-lab at the University Clinic Schleswig-Holstein, Kiel, Germany, and use metabolic modeling to investigate host-microbe interactions. I am trying to understand how metabolic interaction within the microbiome and with the host change under changing environmental conditions and how this can be leveraged for medical applications. Before that, I tried to understand how temperature and microbes change developmental decisions in the freshwater polyp Hydra - still a hidden passion of mine.

:::
::::::


## Current projects

### Modeling host-microbe metabolic interactions in IBD

In this project we model metabolic changes in the microbiome, the gut and the blood of IBD patients and associate these with disease activity. We use metabolic modeling and integrate different omics layer into this framework, including transcriptomics, 16S-amplicon sequencing and metabolomics. With this, we were able to predict crosslinks between microbial metabolism during inflammation and local (gut) and global (blood) metabolism of the patients. Furthermore, we used metabolic modeling to predict dietary interventions which would target these crosslinks and revert changes in the metabolism to a healthy state.

### Metazoan transcriptional responses to microbiome dynamics reveal species-specific adaptations

Animals emerged within microbial environments, we thus hypothesized that they have evolved mechanisms to sense microbes and modulate their own physiology accordingly and that these regulations arose early in the metazoan evolution. To detect and describe these putative mechanisms, we scrutinized eight phylogenetically diverse metazoan species for common patterns of transcriptional regulation associated with response to microbial colonization. We found that species specific adaptation to microbial signals appeared to be predominant. Nevertheless, we detected common signals suggesting microbial control of amino acid metabolism, cell motility and epithelial differentiation.

### Evolutionary conserved metazoan associated microbial genetic content (PI)

We gathered metagenomic data sets from diverse metazoans and reconstructed metagenomic assembled genomes for over 4k different bacteria and published these in a data base: [MetMicDB](https://www.metmicdb.uni-kiel.de/). We want to understand how diverse different microbial communities are in different hosts, or whether we can detect common evolutionary patterns in these communities - either on the genetic or metabolic level.

### Intervention induced microbial dynamics in the *Hydra* microbiome (PI)

Metabolic interactions are the most basic forms of interactions between bacteria, yet we do not know how much these metabolic interactions govern the assembly of microbial communities. Here we try assess the contribution of metabolic interactions to changes in the microbial community. To this end, we use *Hydra* as a model system, due to its simple microbiome (~10-20 species) and its easy tractability in the laboratory. We will use metabolic models to understand the metabolic interactions between the bacteria in the microbiome in *Hydra*
 after exposure to different metabolites. We use the data to predict microbial dynamics observed in the lab and additionally try to estimate the contribution of the host to microbial changes.

### Building community models from different sources (PI)

In this project we develop a software package to combine metabolic models of different origin to a community model which can serve to model metabolic interactions between organisms represented by the input models. In metabolic modeling we suffer from a lack of standardization and several databases exist which describe metabolism in terms of reactions and metabolites. Yet, these databases are not natively compatible with each other, leading of to the problem that common namespaces and conversion reactions have to be generated manually, if one would like to model metabolic interactions between these models. The final software package aims to lift this burden and should eventually provide an (semi-) automatic way to combine models using different namespaces.

# Teaching

[Ecology and Evolution](teaching/EcologyAndEvolution.html)

# Contact

+ Mail: [j.taubenheim@iem.uni-kiel.de](mailto:j.taubenheim@iem.uni-kiel.de)
+ Mastodon: [\@JanTaubenheim](https://mstdn.science/@JanTaubenheim)
- ORCID: [0000-0001-7283-1768](https://orcid.org/0000-0001-7283-1768)

You can find my github account [here](https://github.com/porthmeus) - here I run under the name Porthmeus.

# Vacancies

There are currently no free vacancies - however if you are interested in working with me/us please do not hesitate to get in contact. 

We are always looking for bachelor/master students and we offer [different projects](https://docs.google.com/document/d/11_YLcS413hnhSAfcMI3ouTJTWMBxgQ-ctiOiBYLEm1A/edit) but are open to new project suggestions alike. 

If you are looking for a PhD student or a Postdoc position and you want to work with us - get into contact. There is usually always a possibility to work together and to find financial support, if needed.

# CV

<table>
<tbody>
  <tr>
    <td class="tg-uzvj">Since 10/2020</td>
    <td class="tg-uzvj">Research Fellow</td>
    <td class="tg-uzvj">Institute for Experimental Medicine, Medical Systems Biology, University Clinic Schleswig-Holstein, Kiel, Germany</td>
  </tr>
  <tr>
    <td class="tg-lboi">03/2019-09/2020</td>
    <td class="tg-lboi">Research Fellow</td>
    <td class="tg-lboi">Institute for Zoology and Organismic Interaction, Heinrich Heine University Düsseldorf, Germany</td>
  </tr>
  <tr>
    <td class="tg-lboi">10/2013-12/2018</td>
    <td class="tg-lboi">PhD student &amp; Research Fellow</td>
    <td class="tg-lboi">Institute for Zoology, Christian-Albrechts-University Kiel, Germany</td>
  </tr>
  <tr>
    <td class="tg-lboi">10/2011-09/2013</td>
    <td class="tg-lboi">Master Medical Biology</td>
    <td class="tg-lboi">University Duisburg-Essen, Germany</td>
  </tr>
  <tr>
    <td class="tg-cly1">10/2008-09/2011</td>
    <td class="tg-cly1">Bachelor Medical Biotechnology</td>
    <td class="tg-cly1">University Rostock, Germany</td>
  </tr>
</tbody>
</table>

## Acquired funding and academic distinctions

+ Individual Research Grant (DFG) – “Understanding Environment-Microbiome-Host metabolic interactions in Hydra” (2023)
- Young Investigator Award 2022 – CRC 1182 “Origin and function of Metaorganisms” (2022)
- Walter-Benjamin Fellowship (DFG) – but not accepted due to personal reasons (2020)


# Publications

[See my Google Scholar profile](https://scholar.google.com/citations?user=B5ejjmIAAAAJ&hl=de&oi=ao)
